python/cudf/cudf/tests/test_compile_udf.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+from cudf.utils import cudautils
+from numba import types
+
+
+def setup_function():
+    cudautils._udf_code_cache.clear()
+
+
+def assert_cache_size(size):
+    assert cudautils._udf_code_cache.currsize == size
+
+
+def test_first_compile_sets_cache_entry():
+    # The first compilation should put an entry in the cache
```

Comment:
I just love how well documented your tests are!

Reply:
I also appreciate the comments!

You can use triple quotes if you want the `help` function to work: https://stackoverflow.com/questions/2357230/what-is-the-proper-way-to-comment-functions-in-python

Reply:
@esoha-nvidia Thanks for the pointer - I generally don't do this for comments describing tests because I find that some test runners print out the first line of the help instead of the test name, which makes it more difficult to see at a glance which tests are running / have run recently.

Reply:
Didn't know about that.  Okay!  Looks good to me.